src/components/content/order/ServiceList.tsx

```diff
@@ -353,11 +368,36 @@ function ServiceList(): JSX.Element {
         setIsModalOpen(false);
     };
 
+    const handleCancelMigrateModel = () => {
+        refreshData();
+        setCurrentServiceVo(undefined);
+        setIsMigrateModelOpen(false);
+    };
+
+    const getCloseStatus = (isClose: boolean) => {
+        setIsMigrateModelOpen(isClose);
+    };
+
     return (
         <div className={'services-content'}>
             {tip}
             <Modal title={'Service Details'} width={700} footer={null} open={isModalOpen} onCancel={handleCancel}>
-                <MyServiceDetails endPointInfo={content} requestParamsInfo={requestParams} />
+                <MyServiceDetails
+                    endPointInfo={content}
+                    requestParamsInfo={requestParams}
+                    resultMessage={resultMessage}
+                />
+            </Modal>
+            <Modal
```

Comment:
the migrate model is not fitting the page. Have same issue on big and small displays.

Reply:
fixed.